src/rgw/rgw_rest_s3.cc

```diff
@@ -4869,7 +4869,8 @@ bool RGWHandler_REST_S3Website::web_dir() const {
 
   RGWObjectCtx& obj_ctx = *static_cast<RGWObjectCtx *>(s->obj_ctx);
   obj_ctx.set_atomic(obj);
-  obj_ctx.set_prefetch_data(obj);
+  prefetch_range prefetch{0, static_cast<uint64_t>(s->cct->_conf->rgw_max_chunk_size)};
```

Comment:
`RGWRados::raw_obj_stat()` is already passing in `rgw_max_chunk_size`, this should use the default-constructed `prefetch_range` from 0-max

same with `get_obj_head()`

Reply:
Done